### PR TITLE
a4polls/poll_dashboard: adding id and new a4 hash

### PR DESCRIPTION
**Pull request by phillimorland**
…ger reload - not working

@fuzzylogic2000 @rmader at some point I could do with a hand with this but probably not today

### CODE REVIEW

File: euth_wagtail/templates/a4polls/poll_dashboard.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends "a4dashboard/base_dashboard_project.html" %}
+{% load i18n react_polls %}
+
+{% block dashboard_project_content %}
+    <h1 class="u-first-heading">{% trans 'Edit poll' %}</h1>
+
+    {% react_poll_form poll True %}
```
Comment (fuzzylogic2000) [inline euth_wagtail/templates/a4polls/poll_dashboard.html:7]: I actually have no idea. The True should reload it, and it's also included like that in a4, soooo, hmmm.

Comment (phillimorland) [reply euth_wagtail/templates/a4polls/poll_dashboard.html:7]: well good to know that's what you meant! :) it's included twice in mB, but that is a different structure, i feel like this is a morning brain thing so will do bootstrap issues for rest of day and come back to this in morning :)

**Comment by rmader** [comment]
Ok so the issue with the poll reloading is actually not the template - we do, like a+, successfully call into `updateDashboard` (https://github.com/liqd/adhocracy4/blob/master/adhocracy4/dashboard/assets/dashboard.js#L9). The problem appears to be that only elements with the classes `l-menu__menu` and `l-menu__aside` get updated - and apparently we're not using those classes in the opin templates

File: euth_wagtail/templates/a4polls/poll_dashboard.html (hunk 1 above)
Comment (rmader) [reply euth_wagtail/templates/a4polls/poll_dashboard.html:7]: This commit can be dropped - the a4 template works just fine

**Comment by phillimorland** [comment]
@rmader ahhhhh that makes sense

**Comment by fuzzylogic2000** [comment]
Yeah, makes sense, but doesn't seem like the best class to use. Can't we have an id specially for the dashboard update or something like that?

**Comment by phillimorland** [comment]
yep! i'll do that now

**Comment by phillimorland** [comment]
@fuzzylogic2000 yes!

**Comment by rmader** [comment]
Oh please wait another sec

**Comment by rmader** [comment]
There's one nit: https://www.w3schools.com/html/html_id.asp states:
> The id attribute specifies a unique id for an HTML element (the value must be unique within the HTML document).

So if I understand that right we need to use a class instead :/

**Comment by phillimorland** [comment]
@rmader i disagree with using class's for js selecting but if we are going to do this intentionally then we should decide on a naming convention before we implement to ensure they are identifiable. Camel case we already do this with react and don't we use snake case for python? what others are there? @fuzzylogic2000 @Rineee any ideas, preferences?

**Comment by fuzzylogic2000** [comment]
Oh, yeah, right, an id should only be used once, so we need to use a class. :(
That's annoying because we could have kept the classes for styling and used the id for other things like that. Maybe we can use update_dashboard as the class instead then, so that at least the naming of the class makes sense and is not also used for styling?
I don't care which case we use for this. Maybe not camel. With BEM we want to use this weird snake-kebap-case mix for the classes anyway, right?

**Comment by phillimorland** [comment]
I think we should use something that can't be mistaken for a styling class as it leads to confusion like we had before

**Comment by fuzzylogic2000** [comment]
You're right! Is update_dashboard clear enough or should we use CamelCase then to make it totally clear?

**Comment by rmader** [comment]
Maybe something like `js-selector-update-dashboard`? Naming things... :(

**Comment by phillimorland** [comment]
naming things sucks! ok js-selector wins as will ensure no confusion :p

File: euth_wagtail/templates/a4dashboard/base_dashboard_project.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <h2 class="dashboard-left-nav-header">{% trans 'Module'%}: {{ module_menu.module
             {% block dashboard_project_content %}{% endblock %}
         </div>
 
-        <div class="col-md-3">
+        <div id="update_dashboard" class="col-md-3">
```
Comment (rmader) [inline euth_wagtail/templates/a4dashboard/base_dashboard_project.html:41]: This was left out ;)

Comment (rmader) [reply euth_wagtail/templates/a4dashboard/base_dashboard_project.html]: Took the freedom to fix that

**Comment by rmader** [comment]
Awesome!

File: euth_wagtail/templates/a4dashboard/base_dashboard_project.html (hunk 2 above)
Comment (phillimorland) [reply euth_wagtail/templates/a4dashboard/base_dashboard_project.html]: thank you! :)

**Pull request merged**